bin/wp-cli.php

```diff
@@ -82,6 +84,35 @@ public function flush() {
 		}
 	}
 
+ 	/**
+	 * Add date range when retrieving posts in bulk.
+	 * Dates need to be passed as mmddyyy. See synopsis for index function.
+	 *
+	 * @param $args array
+	 * @return $args array
+	 */
+	public function __apply_date_range( $args ) {
+		$args['date_query'] = array(
+			0 => array(
+				'inclusive' => true,
+			),
+		);
+		if ( isset ( $this->date_range['from'] ) ) {
+			$args['date_query'][0]['after'] = array(
+				'year'  => substr( $this->date_range['from'], -4 ),
```

Comment:
Instead of using substr, you should use `strtotime()` and then use date functions to set these args. Also, are these inclusive? You should document whether or not they are.